src/transformers/models/tapas/modeling_tf_tapas.py

```diff
@@ -830,7 +838,6 @@ class TFTapasMainLayer(keras.layers.Layer):
     config_class = TapasConfig
 
     def __init__(self, config: TapasConfig, add_pooling_layer: bool = True, **kwargs):
-        requires_backends(self, "tensorflow_probability")
```

Comment:
Also I am not sure this is the best action to take.

Reply:
same as above the is available is not always write